Node origin
Node destination
string ACTION_DRIVE="DRIVE"
string ACTION_OPEN_DOOR=="OPEN_DOOR"
string ACTION_PUSH_OBJECT="PUSH_OBJECT"
#string ACTION_USE_ELEVATOR="USE_ELEVATOR"
string action_type  # Any of ACTION_*
float32 cost  # How expensive is is to travel along this edge